mimesis/providers/path.py

```diff
@@ -87,5 +87,5 @@ def project_dir(self, user_gender='female'):
             /home/sherika/Development/Falcon/mercenary
         """
         project = self.random.choice(PROJECT_NAMES)
-        return os.path.join(
-            self.dev_dir(user_gender), project)
+        path = '\\' if self.platform == 'win32' else '/'
+        return (self.dev_dir(user_gender) + '{}' + project).format(path)
```

Comment:
Oh, now I see. It is not `return (...)`, it is some complex expression hidden there. 
It is better to reduce the complexity here.